enterprise/backend/test/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions_test.clj

```diff
@@ -1031,7 +1031,7 @@
                    (run-query)))))))))
 
 (deftest persistence-disabled-when-sandboxed
-  (mt/test-driver :postgres
+  (mt/test-drivers (mt/normal-drivers-with-feature :persist-models)
```

Comment:
adding extra drivers but forgetting to set `mt/test-drivers` with an s makes tests silently not run. See https://github.com/metabase/metabase/issues/28735